src/glue/jobs/json_to_parquet.py

```diff
@@ -332,13 +413,141 @@ def write_table_to_s3(
             format = "parquet",
             transformation_ctx="write_dynamic_frame")
 
+class CountEventType(Enum):
+    """The event associated with a count."""
+
+    """
+    This table has just now been read from the Glue table catalog
+    and has not yet had any transformations done to it.
+    """
+    READ = "READ"
```

Comment:
The docstrings need to go below the thing it's referencing: https://peps.python.org/pep-0224/

If it does not than the hover document is incorrect:
![image](https://github.com/Sage-Bionetworks/recover/assets/17128019/1210865b-19cc-49ad-8b25-5691d6cb8c20)

Reply:
Oh, that's an interesting choice.